fourkitchens/suhumsci
# shs-4959: adjusts size of media thumbnail in ck5

## Summary
Adjusts media size for CK5 editor

## Steps to Test
1. Go to a flexible page. Add some text. Click the media button and add a YouTube video.
2. The video should be larger than the current much smaller thumbnail.
3. Do the same for an image.

## Screenshots
![Screenshot 2023-06-13 at 10 10 57 AM](https://github.com/fourkitchens/suhumsci/assets/8405274/1d467314-f5b7-41d8-a0b0-38057f945be0)

## Notes
- I set the aspect ratio of videos to `16/9` but some embedded videos that come from Vimeo do not follow that ratio. It displays fine (no stretching) just isn't as big as other videos/YouTube
- CK5 has an included `max-width: 50%` for media which I kept. I think it looks much better than the over the top large size that currently happens with media in the old CK.
